### PR TITLE
Profile screenshots (EXPOSUREAPP-12673)

### PR DESCRIPTION
**Pull request by SamuraiKek**
Adds screenshots for rapid profile list.

### CODE REVIEW

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListFragmentTest.kt [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                        "BDAY:19690420\n" +
+                        "EMAIL;TYPE=home:max.musterman@gmail.com\n" +
+                        "TEL;TYPE=\"cell,home\":(359) 260-345\n" +
+                        "ADR;TYPE=home:;;Long Str\\, No. 4;Liverpool;;12345\n" +
+                        "REV:20220418T091325Z\n" +
+                        "END:VCARD",
+                    onClickAction = { _, _ -> }
+                ),
+
+                ProfileCard.Item(
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListFragmentTest.kt:82]: Isn't there some female dummy person to use here? I see Andrea Schneider on figma.

Comment (SamuraiKek) [reply Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListFragmentTest.kt:82]: Yep, I'll adapt it. I just copy pasted this profile data from another screenshot test 😄

Comment (chiljamgossow) [reply Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListFragmentTest.kt:82]: thanks

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListFragmentTest.kt [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                    profile = Profile(
+                        id = 2,
+                        firstName = "Bax",
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListFragmentTest.kt:85]: you could use Max Mustermann here then :)

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListFragmentTest.kt [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                        email = "baxbustermann@web.de"
+                    ),
+                    qrCode = "BEGIN:VCARD\n" +
+                        "VERSION:4.0\n" +
+                        "N:Busterman;Bax;;;\n" +
+                        "FN:Bax Busterman\n" +
+                        "BDAY:19690420\n" +
+                        "EMAIL;TYPE=home:max.musterman@gmail.com\n" +
+                        "TEL;TYPE=\"cell,home\":(359) 260-345\n" +
+                        "ADR;TYPE=home:;;Long Str\\, No. 4;Liverpool;;12345\n" +
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListFragmentTest.kt:101]: Does the qr code data need to match? I thought they are picky about screen shots as they provide them to the public

Comment (SamuraiKek) [reply Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListFragmentTest.kt:101]: No, I don't think so, I just tried to keep it consistent.

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListFragmentTest.kt (hunk 2 above)
Comment (SamuraiKek) [reply Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/coronatest/rat/profile/list/ProfileListFragmentTest.kt:85]: Alright :) Say goodbye to my friend Bax 😆

**Review by CV113 — DISMISSED** [review]
UA approved

**Comment by jurajkusnier** [comment]
I have an issue with running the screenshot on my device
image

**Review by jurajkusnier — DISMISSED** [review]

**Review by jurajkusnier — APPROVED** [review]

**Review by chiljamgossow — APPROVED** [review]

**Pull request merged**